Kotlin-compose/build.sh

```diff
@@ -4,6 +4,7 @@ set -eo pipefail
 
 # Cleanup old files.
 rm -rf build/
+rm -rf compose-multiplatform/
```

Comment:
drive by fix?

Reply:
Yes, makes clean rebuilds easier, since you don't need to remove the source checkout manually